queries.sql

```diff
@@ -7,4 +7,74 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight >= 10.4 AND weight <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight >= 10.4 AND weight <= 17.3;
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN;
+
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+COMMIT;
+
```

Comment:
- [ ] Same here  Before the `COMMIT` make sure to check that the changes were made, then after check the changes were undone


```sql
SELECT species from animals; 
   COMMIT;
SELECT species from animals;
```